services/ymax-planner/src/main.ts

```diff
@@ -87,8 +87,16 @@ export const main = async (
     clusterName,
     alchemyApiKey: config.alchemyApiKey,
   });
+  const now = Date.now;
 
-  const powers = { evmCtx, rpc, spectrum, cosmosRest, signingSmartWalletKit };
+  const powers = {
+    evmCtx,
+    rpc,
+    spectrum,
+    cosmosRest,
+    signingSmartWalletKit,
+    now,
```

Comment:
There's no need to bind a single-use reference to a variable name.
```suggestion
    now: Date.now,
```